mne/viz/_3d.py

```diff
@@ -415,6 +454,19 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         View to use. See surfer.Brain().
     colorbar : bool
         If True, display colorbar on scene.
+    limits : str | 3-tuple | dict
```

Comment:
Yea, good point. I guess I could go either way here. It's basically philosophical issue: do we want to make everyone conform to 1 API or make it more of a catch-all and leave more options open to the user?

Thoughts? @agramfort, @dengemann